amaranth/vendor/xilinx.py

```diff
@@ -343,14 +343,20 @@ def vendor_toolchain(self):
 
     _symbiflow_part_map = {
         "xc7a35ticsg324-1L": "xc7a35tcsg324-1", # Arty-A7
+        "xc7a15tcpg236-1": "xc7a15tcpg236-1", # Cmod-A7-15
+        "xc7a35tcpg236-1": "xc7a35tcpg236-1", # Cmod-A7-35
     }
 
     _symbiflow_bitstream_device_map = {
             "xc7a35ticsg324-1L": "artix7",
+            "xc7a15tcpg236-1": "artix7",
+            "xc7a35tcpg236-1": "artix7",
     }
 
     _symbiflow_device_map = {
             "xc7a35ticsg324-1L": "xc7a50t_test",
+            "xc7a15tcpg236-1": "xc7a50t_test",
+            "xc7a35tcpg236-1": "xc7a50t_test",
```

Comment:
This one seems fine though.

Reply:
Here a prefix match should also be able to do the job. The used architecture definition does not depend on package or speed grade afaict.

Reply:
I've added here whatever is also supported in f4pga project.